server/src/models/Trail.js

```diff
@@ -5,6 +5,34 @@ class Trail extends Model {
     return "trails"
   }
 
+  static get relationMappings() {
+    const { Review, User } = require("./index")
+
+    return {
+      reviews: {
+        relation: Model.HasManyRelation,
+        modelClass: Review,
+        join: {
+          from: "trails.id",
+          to: "reviews.trailId",
+        },
+      },
+
+      user: {
+        relation: Model.BelongsToOneRelation,
```

Comment:
A few things going on here.
* Line 22 says this is a `BelongsToOneRelation`, but there is a `has many through` set up that is used for a `many-to-many` association.  This would return the users who wrote reviews for this trail.  Is that the goal here?